src/compile/mark/mark.ts

```diff
@@ -182,8 +182,9 @@ export function pathGroupingFields(mark: Mark, encoding: Encoding<string>): stri
       case 'text':
       case 'shape':
         return details;
-
       case 'tooltip':
+        // tooltip fields should not be added to group by
+        return details;
```

Comment:
Do we need a separate `case`/`return` here, or could `tooltip` be moved to the previous block?